plugins/woocommerce/changelog/43414-product-collection-hide-products-beta-from-inserter

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: update
+
+Products (Beta): hide block from inserter. Use Product Collection instead
```

Comment:
Nitpick: I believe we can enhance this changelog message.

Reply:
> Nitpick: I believe we can enhance this changelog message.

Sure, what do you suggest? I'm not sure what aspect you have in mind 😅

Reply:
I just didn't liked `dot` in the middle of the sentence 😅

Maybe we can rephrase it:
```
Products (Beta): hide block from inserter in favor of Product Collection block
```

But it was a nitpick, so feel free to ignore it 😅

Reply:
I see and I like that! I updated the changelog.